SharpBucket/SharpBucket.cs

```diff
@@ -11,7 +11,11 @@ namespace SharpBucket{
     /// </summary>
     public class SharpBucket{
         private Authenticate authenticator;
-        protected string _baseUrl;
```

Comment:
We should change this in all files or on none. Otherwise it will just get quite inconsistent. Lets not change it for now and maybe do a PR just for that.